tensorflow/core/graph/mkl_graph_util.h

```diff
@@ -151,8 +151,9 @@ inline string GetMklNativeOpName(const string& name) {
   // prefixed with _Mkl instead of _MklNative.
   bool result =
       (0 == name.compare("ConjugateTranspose") ||
-       0 == name.compare("BatchMatMul") || 0 == name.compare("BatchMatMulV2") ||
-       0 == name.compare("MatMul") || 0 == name.compare("Transpose"));
+       0 == name.compare("BatchMatMul") || 0 == name.compare("Einsum") ||
+       0 == name.compare("BatchMatMulV2") || 0 == name.compare("MatMul") ||
```

Comment:
Nit: Can we keep `BatchMatMul` and `BatchMatmulV2` together and put `Einsum` after `BatchMatMulV2`?
```suggestion
       0 == name.compare("BatchMatMul") || 0 == name.compare("BatchMatMulV2") ||
       0 == name.compare("Einsum") || 0 == name.compare("MatMul") ||
```